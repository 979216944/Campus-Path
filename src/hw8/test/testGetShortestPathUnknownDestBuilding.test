# Verify that the program prints one unknown building
# with unknown destination building.

# get shortest path with unknown destination building
r
CSE
unknown

# quit
q