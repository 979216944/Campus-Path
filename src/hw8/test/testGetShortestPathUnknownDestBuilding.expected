Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program prints one unknown building
# with unknown destination building.

# get shortest path with unknown destination building
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: unknown

Enter an option ('m' to see the menu):
# quit
